render-test/tests/should-fail.json

```diff
@@ -2,5 +2,7 @@
   "tests/file-size/fail-size-is-over": "Should fail, size is bigger than expected.",
   "tests/file-size/fail-size-is-under": "Should fail, size is smaller than expected.",
   "tests/file-size/fail-file-doesnt-match": "Should fail, doesn't match the expectation.",
-  "tests/file-size/fail-file-not-found": "Should fail, file not found."
+  "tests/file-size/fail-file-not-found": "Should fail, file not found.",
+  "tests/network/fail-requests": "Should fail, number of requests higher than expected.",
+  "tests/network/fail-transferred": "Should fail, amount of transferred data higher than expected."
```

Comment:
Maybe add a test for when count and transferred is greater/lower.

Reply:
Done